File: src/main/java/no/nav/service/pensjon/regler/to/BeregnYtelseResponse.java
package no.nav.service.pensjon.regler.to;

import java.util.ArrayList;
import java.util.List;
import java.util.Vector;

import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.beregning.Beregning;

/**
 * Klassen representerer hele resultatet fra et kall til en regeltjeneste for ytelsesberegning.
 */
public class BeregnYtelseResponse extends ServiceResponse {

	/**
	 * Pakkseddel for beregningene.
	 */
	private Pakkseddel pakkseddel = new Pakkseddel(true, true);
	/**
	 * En liste av beregninger som utgjør resultatet.
	 */
	private List <Beregning> beregningsListe = new Vector<Beregning>();


}

File: src/main/kotlin/no/nav/pensjon/regler/to/BeregnYtelseResponse.kt
package no.nav.pensjon.regler.to


import no.nav.pensjon.regler.domain.beregning.Beregning
import java.util.*

/**
 * Klassen representerer hele resultatet fra et kall til en regeltjeneste for ytelsesberegning.
 */
class BeregnYtelseResponse : ServiceResponse() {
    /**
     * En liste av beregninger som utgjør resultatet.
     */
    var beregningsListe: List<Beregning> = Vector()

}